### ISSUE TITLE
Vercel deployment in CI is failing with "Uncaught AssertionError"

### ISSUE THREAD

**Issue by trivikr** [denoland/doc_website#99]
Error:
```console
error: Uncaught AssertionError: Assertion failed.
    at Object.assert ($deno$/util.ts:33:11)
    at compile ($deno$/compiler.ts:1355:5)
    at async tsCompilerOnMessage ($deno$/compiler.ts:1548:22)
    at async workerMessageRecvCallback ($deno$/runtime_worker.ts:74:9)
    at buildDenoLambda (/zeit/5414d56625280cb6/.build-utils/.builder/node_modules/now-deno/dist/index.js:70:15)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
```

Refs:
* PR https://github.com/denoland/doc_website/pull/98
  * Deployment failure https://vercel.com/denoland/doc-website/838yvfwd1
* PR https://github.com/denoland/doc_website/pull/93
  * Deployment failure https://vercel.com/denoland/doc-website/838yvfwd1

**Comment by lucacasonato** [comment]
I am working on it - sorry

**Comment by lucacasonato** [comment]
Fixed in #102
